textalyze.js

```diff
@@ -14,10 +14,32 @@
 function itemCounts(array) {
   let counts = new Map();
 
+  array.forEach(item => {
+
+    if (!counts.has(item)) {
+      counts.set(item, 0);
+    }
+
+    counts.set(item, parseInt(counts.get(item)) + 1);
```

Comment:
There's no need for `parseInt` here.

The `counts` variable is defined in this function and only in this function, so you have complete control over what types of values it has.

Since you _know_ the values will only ever be integers, you don't need `parseInt`.